src/index.js

```diff
@@ -1,6 +1,7 @@
 // debug output
 let __DEBUG__
 function debug (...args) {
+  /* istanbul ignore if */
   if (__DEBUG__) {
     if (!console.group) {
       args.unshift('%credux-undo', 'font-style: italic')
```

Comment:
Not sure if this is a good idea.. but those will never be covered since that would cause a lot of console logs in the tests..